definitions/npm/@reach/router_v1.1.x/flow_v0.63.x-/router_v1.1.x.js

```diff
@@ -1,10 +1,17 @@
 // @flow
 
 declare module '@reach/router' {
-  declare type NavigateFn = (to: string, options?: NavigateOptions<{}>) => void;
+  declare type DomLocation = typeof location;
```

Comment:
You could perhaps just `declare type DomLocation = Location;`, see

https://github.com/facebook/flow/blob/master/lib/bom.js#L303
https://github.com/facebook/flow/blob/master/lib/bom.js#L320

Reply:
I can't because definition already has own `Location`

Reply:
https://github.com/flow-typed/flow-typed/blob/8b0a1f6a0147971ddc60f872f7cbdd37cdfdccab/definitions/npm/%40reach/router_v1.1.x/flow_v0.63.x-/router_v1.1.x.js#L93-L95